ssh.md

```diff
@@ -1,26 +1,65 @@
-#SSH
-**SSH** permit you to command remotly your sever by command line interface (CLI).
+# SSH
 
-##### To connect to your server:
+## What's SSH ?
 
-`ssh admin@my-server.org`
+**SSH** stands for Secure Shell, and refers to a protocol that allows to remotly control a machine using the command line interface (CLI). It is available by default on Linux and Mac OS / OSX. On Windows, unfortunately you might need to use the [PuTTy](http://www.fastcomet.com/tutorials/getting-started/putty) software.
+
+## During Yunohost installation
+
+#### Find your IP
+
+If you are installing on a VPS, then your VPS provider should have gave you your IP address. 
+
+If you are installing at home (e.g. on a Raspberry Pi or OLinuXino), then you need to find out which IP has been attributed to your board after you plugged it to your internet box / router. Several ways exists to find your server's IP :
+
+- open a terminal and use `sudo arp-scan --local` to list the IP on your local network ;
+- use your internet box / router interface to list the DHCP clients ;
+- plug a screen on your server, log in and type `ifconfig`.
```

Comment:
"look into the logs of your rooter"
"you can also uses `nmap -sP $local-subnet/24` to see all conected ips and test them [advanced]"